static/cli_download.html

```diff
@@ -18,7 +18,7 @@
   <div id="topnav" class='topnav'>
     <a href="index.html">HOME</a>
     <a href="calculate_score.html">CALCULATE</a>
-    <a href="cite.html">CITE</a>
+    <a href="studies.html">STUDIES</a>
```

Comment:
I think the change probably needs to be made on the index.html page and the visualize.html page as well.

Reply:
Done :)